lib/pack.js

```diff
@@ -96,11 +110,10 @@ module.exports = {
           });
 
           cli.log(output);
-		  return resolve();
+          return resolve();
         } catch (err) {
           console.error('An error occured while restoring packages');
-          console.error(err.stdout.toString('utf8'));
-          console.error(err.toString('utf8'));
+          console.error(err);
```

Comment:
Why did you changed this ?

Reply:
It still looks changed

Reply:
@karnafun please check this

Reply:
I just copy pasted the exact same code, reverting any changes done to console.error in the code.